src/routes/transactions/entities/swap-order-info.entity.ts

```diff
@@ -51,12 +51,16 @@ export abstract class SwapOrderTransactionInfo extends TransactionInfo {
   @ApiProperty({ description: 'The timestamp when the order expires' })
   expiresTimestamp: number;
 
+  @ApiProperty({ description: 'The filled percentage of the order' })
```

Comment:
Out of the scope of this PR but what do you think about adding `example` to these? I understand the value from the PR description, but maybe not in a few months 🙈

Reply:
Good point! Added examples in [7c939f5](https://github.com/safe-global/safe-client-gateway/pull/1338/commits/7c939f5a5ebab7af689bd6e372923f188ce5a215)